src/multivariate/mvnormal.jl

```diff
@@ -59,42 +59,37 @@ _pdf!(r::AbstractArray, d::AbstractMvNormal, x::AbstractMatrix) = exp!(_logpdf!(
 #   Multivariate normal distribution with mean parameters
 #
 ###########################################################
-immutable MvNormal{T<:Real,Cov<:AbstractPDMat,Mean<:Union{Vector, ZeroVector}} <: AbstractMvNormal
+immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector, ZeroVector}} <: AbstractMvNormal
     μ::Mean
     Σ::Cov
 end
 
 const MultivariateNormal = MvNormal  # for the purpose of backward compatibility
 
-typealias IsoNormal  MvNormal{Float64,ScalMat{Float64},Vector{Float64}}
-typealias DiagNormal MvNormal{Float64,PDiagMat{Float64,Vector{Float64}},Vector{Float64}}
-typealias FullNormal MvNormal{Float64,PDMat{Float64,Matrix{Float64}},Vector{Float64}}
+typealias IsoNormal  MvNormal{ScalMat{Float64},Vector{Float64}}
```

Comment:
I think this might be reverting @jmxpearson's recent changes to `MvNormal`. I think it is necessary to have the element type as a parameter.